package.json

```diff
@@ -12,19 +12,20 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "grunt": "~0.4.1",
     "coffee-script": ">= 1.6",
+    "grunt": "~0.4.1",
+    "grunt-build-gh-pages": "0.0.4",
+    "grunt-contrib-clean": "~0.4.1",
     "grunt-contrib-coffee": "~0.6.4",
-    "grunt-contrib-uglify": "~0.2.0",
+    "grunt-contrib-compass": "~0.5.0",
     "grunt-contrib-concat": "~0.1.3",
-    "grunt-contrib-watch": "~0.3.1",
     "grunt-contrib-copy": "~0.4.1",
-    "grunt-contrib-clean": "~0.4.1",
-    "grunt-contrib-compass": "~0.5.0",
     "grunt-contrib-cssmin": "~0.6.1",
-    "grunt-build-gh-pages": "0.0.4",
+    "grunt-contrib-uglify": "~0.2.0",
+    "grunt-contrib-watch": "~0.3.1",
+    "grunt-dom-munger": "~2.0.1",
     "grunt-zip": "~0.9.2",
-    "grunt-dom-munger": "~2.0.1"
+    "load-grunt-tasks": "^3.0.0"
```

Comment:
This reordering happened because this package.json was built by hand. When you use the CLI it sorts the packages alphabetically, automatically.